canonical-web-and-design/vanilla-framework
# merge article pagination and pagination, improve spacing

## Done

- Ensure pagination chevrons can use text labels without causing spacing issues
- Refactor pagination buttons to use the button placeholders instead of separate styling
- Remove the article pagination page, and replace it with a mention on the pagination component documentation
- Update the doc copy to explain when to use pagination, and when to use article pagination

Fixes #3165 #3167

## QA

- Pull code
- Run `./run`
- Open /docs/patterns/pagination, proofread.
- Open /docs/examples/patterns/pagination/pagination-verbose, paste the contenrt of this [pastebin](https://pastebin.canonical.com/p/BcCdX8q5zd/plain/). Ensure spacing matches the icon button spacing

## Screenshots

Before:

![image](https://user-images.githubusercontent.com/2741678/87418174-2f3f5700-c5c9-11ea-95ef-6ae4fe41fadc.png)

After:

![image](https://user-images.githubusercontent.com/2741678/87418306-5ac24180-c5c9-11ea-8c70-99ad0c46d017.png)
